4/README.md

```diff
@@ -1,20 +1,41 @@
-Docker image for Varnish Cache 4.0 HTTP reverse proxy
+Varnish Cache 4.0 HTTP reverse proxy Docker image
 =====================================================
 
-This repository contains the source for Varnish Cache 4.0 Docker image.
+This container image includes Varnish 4.0 server and a reverse proxy for OpenShift and general usage.
```

Comment:
I found that here is just `Varnish 4.0` instead of `Varnish Cache 4.0` is that a problem which needs a fix?